cinema/serializer.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(required=True, max_length=255)
```

Comment:
https://www.django-rest-framework.org/api-guide/fields/#required
it's the default value

Reply:
you can remove it